<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStu</class>
 <widget class="QDialog" name="addStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>addStu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加学生</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,8">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>学号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputId"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,4,4">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonMale">
       <property name="text">
        <string>男</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="buttonFemale">
       <property name="text">
        <string>女</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,8">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>年龄：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="selectAge"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,8">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>院系：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="selectDepartment"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,8">
     <property name="leftMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>兴趣：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBasketball">
         <property name="text">
          <string>篮球</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkVolleyball">
         <property name="text">
          <string>排球</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkSoccer">
         <property name="text">
          <string>足球</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBadminton">
         <property name="text">
          <string>羽毛球</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">insGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,2,2">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="insGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
